docs/getting-started/dockstore-tools.rst

```diff
@@ -1,28 +1,45 @@
 .. note::
-  This tutorial is a continuation of :doc:`Creating a Dockstore Account </getting-started/register-on-dockstore>`. Please complete
-  the tutorial prior to doing this one.
+  This document assumes you have already have a tool ready to register. If you don't, please check :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` first. If you are working with any non-CWL language, you may be looking for :doc:`Registering Workflows on Dockstore </getting-started/dockstore-workflows>` instead of this document.
```

Comment:
Seems odd to reference registering a workflow in a section about getting started with tools.

Reply:
I made that decision intentionally. It is quite possible that someone who does not understand the difference between tools and workflows may end up on this page, so this nudges them in the right direction. It is also an attempt to nudge WDL people into registering workflows instead of tools.

Reply:
Let's be more direct in our language here. I need help with my definition of tool -- should we say it requires a container?

"A tool describes a single command line task. Tools are well-defined and commonly used in the Common Workflow Language (CWL) as they may be chained together to make a workflow. Tools are not explicitly used in other workflow languages (See :doc:`Registering Workflows on Dockstore </getting-started/dockstore-workflows>` instead). In this tutorial, we assume that you have a CWL tool ready to register in Dockstore. "

Reply:
Good to know! What about:

A tool describes a single command line task. Tools are defined in the Common Workflow Language (CWL) and Galaxy where they may be chained together to create a workflow. Dockstore currently only supports CWL tools. In this tutorial, we assume that you have a tool ready to register in Dockstore.